assets/prefabs/structures/dungeon/dungeon3x3StoneCorridor/SandCorridor.prefab

```diff
@@ -0,0 +1,38 @@
+{
```

Comment:
Since all the other files in the two dirs you are adding stuff to start with `dungeon` please add that to the front of your two file names for consistency :-)